README.md

```diff
@@ -99,53 +99,27 @@ After PostgreSQL started up, it's time to adjust some configuration for the quay
 
 ```
 ##### Start psql session in quay-database container
-% podman exec -it quay-database /usr/bin/psql
+% podman exec -it quay-database /usr/bin/psql -d quaydb
```

Comment:
Docs use `psql -d quay -U postgres` - even though this shouldn't matter, we should try to be as close as possible to the official docs.

Reply:
added -U postgres